packages/dds/shared-object-base/src/test/serializer.spec.ts

```diff
@@ -234,16 +248,18 @@ describe("FluidSerializer", () => {
 			);
 
 			// Add some handles to intermediate objects.
-			input.h = handle;
-			input.o1.h = handle;
+			input.h = handle; // eslint-disable-line @typescript-eslint/no-unsafe-member-access
```

Comment:
Can we not use `eslint-disable-next-line` for these (for consistency)?

Reply:
We can. I personally find the resulting code harder to parse than this, but don't feel strongly one way or the other. I tend to prefer "this-line" comments and use next-line only when the line is already too long and reading the disable would require scrolling.

Reply:
I don't feel that strongly, it's just not what we normally do. Feel free to close without action 🙂